src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(setPeople)
+      .catch(() => setError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && <Loader />}
+
+          {error && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+
+          {!people.length && !isLoading && !error && (
+            <p data-cy="noPeopleMessage">There are no people on the server</p>
+          )}
+
+          {people.length && !isLoading && !error && (
```

Comment:
Update condition to not display `0`
```suggestion
          {people.length > 0 && !isLoading && !error && (
```
https://github.com/mate-academy/react_people-table-basics/assets/117587463/3362e7b3-4599-4864-b891-8d5af6414efa

Reply:
Thanks for the video explanation it's very cool and convenient